src/SKUI/css/core.css

```diff
@@ -87,6 +87,21 @@ body.platform-windows
 
 /*******************************************************************************
  *
+ * $BUTTON
+ *
+ ******************************************************************************/
+
+/* undo button padding for the embedded image */
+.control-button img
+{
+  position: absolute;
```

Comment:
Yes indeed, I figured that would be the best "basic setting" because I think most images will be the same size as the button. I thought about additional methods for setting up the image, (position/size/z-index/repetition) but that would make the image some kind of complete sub-object of the button class. The downside for this css-only positioning is that you need to start digging into the css more often.